src/scss/components/breadcrumb/index.scss

```diff
@@ -3,25 +3,28 @@
   align-content: center;
   align-items: center;
   color: $blue-500;
+  color: $rax-component-breadcrumb-base-color;
```

Comment:
Let's use the IE11 fallback pattern below.

Reply:
Changes have been done, please review.